frame/support/src/dispatch.rs

```diff
@@ -18,31 +18,30 @@
 //! Dispatch system. Contains a macro for defining runtime modules and
 //! generating values representing lazy module function calls.
 
-pub use crate::{
-	codec::{
-		Codec, Decode, Encode, EncodeAsRef, EncodeLike, HasCompact, Input, MaxEncodedLen, Output,
-	},
-	scale_info::TypeInfo,
-	sp_std::{
-		fmt, marker,
-		prelude::{Clone, Eq, PartialEq, Vec},
-		result,
-	},
-	traits::{
-		CallMetadata, GetCallIndex, GetCallMetadata, GetCallName, GetStorageVersion,
-		UnfilteredDispatchable,
-	},
+pub use crate::traits::{
+	CallMetadata, GetCallIndex, GetCallMetadata, GetCallName, GetStorageVersion,
+	UnfilteredDispatchable,
 };
+pub use codec::{
+	Codec, Decode, Encode, EncodeAsRef, EncodeLike, HasCompact, Input, MaxEncodedLen, Output,
+};
+pub use scale_info::TypeInfo;
```

Comment:
I wonder how many of these re-exports make sense here, but we can keep that for another issue.  
I found even more in `frame/benchmarking/src/lib.rs` 🙊

Reply:
Most of these exports here are should also be private, but yeah do this later.